build/CallPowerShell.cmd

```diff
@@ -0,0 +1 @@
+PowerShell %~dp0..\%* 
```

Comment:
I see pack.cmd is removed, doesn't look like build.cmd is removed though. Any reason?

Reply:
Good catch, it was an oversight on my part.